core/web/evm_nodes_controller.go

```diff
@@ -58,7 +58,13 @@ func (nc *EVMNodesController) Create(c *gin.Context) {
 		return
 	}
 
-	node, err := nc.App.EVMORM().CreateNode(request)
+	node, err := nc.App.EVMORM().CreateNode(types.Node{
```

Comment:
The `NewNode` types do still add value here, since they map the json field names. So the tradeoff for the ORM simplification is a bit more boiler-plate field copying here, but we don't expect to add more copies of this code since we are moving away from CLI config in the first place, so it is a one-time cost.

We may be able to drop `NewNode`s if we add json tags to the pure `Node` types, but that would be a multi-repo change, so I'd rather leave it out of scope for this PR.